lib/stella/searchable.rb

```diff
@@ -87,7 +88,12 @@ def index_slug
 
       # sets up mappings and settings for index
       def searchable(settings = Stella::Analysis::DEFAULT_SETTINGS, &block)
+        def field(field, opts = {})
+          es_field(field, opts)
+        end
```

Comment:
I think there's no way around having to change the binding of the block, e.g.:

```ruby
class SearchableDSL
  def initialize(model)
    @model = model
  end

  def field(*args)
    @model.es_field(*args)
  end
end

module Searchable
  def searchable(&block)
    SearchableDSL.new(self).instance_eval(&block)
  end
end
```

(Written on my phone, so might have gotten the module's name wrong.)